### ISSUE TITLE
unplayable error with some ids, please check.

### ISSUE THREAD

**Issue by greentornado** [kkdai/youtube#138]
These are some links that playable on the web

https://www.youtube.com/watch?v=gr-IqFcNExY

https://www.youtube.com/watch?v=_4UsZ1icOjA
```
youtubedr download gr-IqFcNExY
cannot playback and download, status: UNPLAYABLE, reason: Video không có sẵn
```

**Comment by greentornado** [comment]
can you check please @corny

**Comment by corny** [comment]
The https://youtube.com/get_video_info endpoint returns the `UNPLAYABLE` status because the uploader has disable embedding of this video on other sites. We need to parse the HTML page in that case like `youtube-dl` does.

**Comment by Julian-Chu** [comment]
> The https://youtube.com/get_video_info endpoint returns the `UNPLAYABLE` status because the uploader has disable embedding of this video on other sites. We need to parse the HTML page in that case like `youtube-dl` does.

@corny   do you know where is the code in youtube-dl for parsing HTML page?

**Comment by kkc** [comment]
maybe we should implement things like this one? https://github.com/ytdl-org/youtube-dl/blob/master/youtube_dl/extractor/youtube.py#L312

**Comment by john-black-3k** [comment]
Please take a look at the PR and let me know what you think.

**Comment by Julian-Chu** [comment]
closed by #144   thanks to @john-black-3k 
release fix version 2.4.1 thanks to @corny
